### PR TITLE
.ci/: export TERM in deps files

### PR DESCRIPTION
**Pull request by harshhx17**
TERM is set in deps files and works for travis ci
However, for it to work in circleci, it needs to be exported.

closes https://github.com/coala/coala-bears/issues/2514

**For short term contributors:** we understand that getting your commits well
defined like we require is a hard task and takes some learning. If you
look to help without wanting to contribute long term there's no need
for you to learn this. Just drop us a message and we'll take care of brushing
up your stuff for merge!

### Checklist

- [ ] I read the [commit guidelines](http://coala.io/commit) and I've followed
      them.
- [ ] I ran coala over my code locally. (*All commits have to pass
      individually.* It is not sufficient to have "fixup commits" on your PR,
      our bot will still report the issues for the previous commit.) You will
      likely receive a lot of bot comments and build failures if coala does not
      pass on every single commit!

After you submit your pull request, **DO NOT click the 'Update Branch' button.**
When asked for a rebase, consult [coala.io/rebase](https://coala.io/rebase)
instead.

Please consider helping us by reviewing other peoples pull requests as well:

- pick up any PR at <https://coala.io/review>
- review it (check <https://coala.io/reviewing> for more info)
- if you are sure that it needs work, use `corobo mark wip <URL>` to get it out
  of the review queue.

The more you review, the more your score will grow at coala.io and we will
review your PRs faster!

### CODE REVIEW

File: .ci/deps.opam.sh [hunk 1]
```diff
@@ -1,7 +1,7 @@
 set -e
 set -x
 
-TERM=dumb
```
[the remark this answers is not in the document]
Comment (harshhx17) [reply .ci/deps.opam.sh]: That would be better
We will just need to add TERM=dumb in travis.yml also
I will do that

**Comment by harshhx17** [comment]
@jayvdb, can you please review this😊

File: circle.yml [hunk 2]
```diff
@@ -1,3 +1,11 @@
+machine:
+  node:
+    version: 6.10.2
```
Comment (jayvdb) [inline circle.yml]: are you 100% sure this needs to be moved from below to here?

Comment (harshhx17) [reply circle.yml]: No, i haven't tested the it the other way😅

**Comment by harshhx17** [comment]
Done @jayvdb :smile:

**Comment by jayvdb** [comment]
ack https://github.com/coala/coala-bears/pull/2515/commits/d47a9005a89cedb55991bf4f769885f19aff5f9d

**Comment by jayvdb** [comment]
@gitmate-bot rebase

**Comment by harshhx17** [comment]
A rebuild passed the tests @jayvdb

**Comment by jayvdb** [comment]
@gitmate-bot ff

**Comment by jayvdb** [comment]
sorry I forgot about this one.

**Pull request merged**